packages/quicktype-core/src/language/TypeScriptZod.ts

```diff
@@ -38,6 +39,13 @@ export class TypeScriptZodTargetLanguage extends TargetLanguage {
         super(displayName, names, extension);
     }
 
+    get stringTypeMapping(): StringTypeMapping {
+        const mapping: Map<TransformedStringTypeKind, PrimitiveStringTypeKind> = new Map();
+        const dateTimeType = "date-time";
```

Comment:
Recommend adding `"date"` format type support as well. `zod.coerce.date()` supports dates without time, and this is also a valid format specified by JSON Schema https://json-schema.org/understanding-json-schema/reference/string.html#dates-and-times